core/server/proxy/src/main/java/alluxio/proxy/s3/signature/AWSCredential.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.proxy.s3.signature;
+
+import alluxio.proxy.s3.S3Exception;
+import alluxio.proxy.s3.S3ErrorCode;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+
+/**
+ * Credential in the AWS authorization header.
+ * Ref: https://docs.aws.amazon.com/AmazonS3/latest/API/
+ * sigv4-auth-using-authorization-header.html
+ *
+ */
+public class AWSCredential {
+  private static final Logger LOG = LoggerFactory.getLogger(AWSCredential.class);
+
+  private String mAccessKeyID;
+  private String mDate;
+  private String mAwsRegion;
```

Comment:
Be consistent with your naming convention. Either use `AWS` in both class and variable names, or `Aws`.